client/js/components/nav.jsx

```diff
@@ -0,0 +1,47 @@
+import React, { Component } from "react"
+import { connect } from "react-redux"
+import Config               from "../config.jsx"
+import { openModal } from '../actions/modal.jsx'
+
+class Nav extends Component {
+
+    constructor() {
+        super()
+    }
+
+    componentDidMount() {
+    }
+
+    render() {
+        return (
+            <div id="nav">
+                <div className="left">The Wall</div>
+                { !this.props.account.logged_in && 
+                    <div className="right">
+                        <div className="tab" onClick={() => this.props.openModal(Config.MODALS.LOGIN)}>LOGIN</div>
+                        <div className="tab" onClick={() => this.props.openModal(Config.MODALS.REGISTER)}>SIGNUP</div>
+                    </div>
+                }
+            </div>
+        )
+    }
+}
+
+function mapStateToProps(store) {
+    return {
+        modal        : store.modal,
```

Comment:
Could the `modal` also be part of the state of the container?

Reply:
I do not know what you mean by this